libraries/default.rb

```diff
@@ -0,0 +1,68 @@
+def collect_github_repositories(github_token, org)
+  if ::ObjectSpace.const_defined?('Chef')
+    # Chef exists! Lets make sure Octokit is installed.
+    chef_gem 'octokit'
+  end
```

Comment:
That makes no sense. Chef is run in order so those functions are being called after line 92. Have you tried removing it and seeing if it works?

Also I think we should move the `require` statements to the top of the file since we are loading `octokit` twice when it is called by the two github functions.